doc/fault-injection.md

```diff
@@ -0,0 +1,152 @@
+# Troubleshooting with Istio: Simulating slow responses from microservices
+
+![Fault Injection](fault-injection.png)
+
+Slow or unresponsive microservices can reveal defects in other system components.  Unhandled timeout
+exceptions can cascade and affect user experience.  Developers get frustrated trying to diagnose, correct,
```

Comment:
Please remove the "Developers get frustrated...", as its distracting and adds little value